tasks/deploy.ts

```diff
@@ -76,48 +70,47 @@ The deployer account used will be the first one returned by ethers.getSigners().
 Therefore, it should be appropriately configured in your Hardhat network configuration.
 This subtask uses the "deploy:impl" subtask for the actual deployment.
 */
-subtask("deploy:module", "Deploys a new module contract")
-    .addParam("module", "SSV Module", null, types.string)
-    .setAction(async ({ module }, hre) => {
-        const moduleValues = Object.values(SSVModules);
-        if (!moduleValues.includes(module)) {
-            throw new Error(`Invalid SSVModule: ${module}. Expected one of: ${moduleValues.join(", ")}`);
-        }
-
-        const moduleAddress = await hre.run("deploy:impl", { contract: module });
-        return moduleAddress;
-    });
-
-task("deploy:token", "Deploys SSV Token")
-    .setAction(async ({ }, hre) => {
-        console.log('Deploying SSV Network Token');
-
-        const ssvTokenFactory = await ethers.getContractFactory('SSVToken');
-        const ssvToken = await ssvTokenFactory.deploy();
-        await ssvToken.deployed();
-
-        console.log(`SSV Network Token deployed to: ${ssvToken.address}`);
-
-    });
+subtask('deploy:module', 'Deploys a new module contract')
+  .addParam('module', 'SSV Module', null, types.string)
+  .addParam('contract', 'SSV Module Contract', null, types.string)
```

Comment:
@Mohsen-T I understand the level of flexibility you want to provide here, but I think this overcomplicates the process of updating modules. Generally, when updating modules for all cases from bugs to new features, we will keep the same version over all envs, from stage (dev) to mainnet.
The reason to use a different contract name when upgrading SSVNetwork or SSVNetworkViews is that for some environments, we need to update state variables via `initializer` function, but only in one environment for testing purposes.
I'm open to hearing a reason to keep the `contract` parameter here, if not, please remove it. thanks!

Reply:
removed SSVStorageV2.sol and contract parameter in update module script